tests/runner.py

```diff
@@ -306,7 +306,7 @@ def is_wasm_backend(self):
   def check_dlfcn(self):
     if self.get_setting('ALLOW_MEMORY_GROWTH') == 1 and not self.is_wasm():
       self.skipTest('no dlfcn with memory growth (without wasm)')
-    if self.get_setting('WASM2JS'):
```

Comment:
Shouldn't this be the same?

Reply:
We decided that WASM=0 is the way to set it on the command line. So we set that in the test mode. Emcc then sets the internal flag WASM2JS to 1 when emcc runs, but the test runner doesn't see that.